### PR TITLE
toggle navbar depending on screen width

### PR DESCRIPTION
**Pull request by sarahk1280**

### CODE REVIEW

File: src/components/Header/index.js [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
+import { size } from '../_size';
 
-const Header = () => (
-  <header>
-    <DesktopNavbar />
-    <Slideshow />
-  </header>
-);
+const Header = () => {
+  const [smallScreen, setSmallScreen] = useState(false);
```
Comment (zzandland) [inline src/components/Header/index.js]: toggleMobile?

File: src/components/Header/index.js [hunk 2]
```diff
@@ -1,15 +1,32 @@
...
+      setSmallScreen(false);
+    }
+  };
+  // adds event listener when component mounts and removes when unmounts
+  useEffect(() => {
+    updateNavbarWidth();
+    window.addEventListener('resize', updateNavbarWidth);
+    return (() => {
+      window.removeEventListener('resize', updateNavbarWidth);
+    });
```
Comment (zzandland) [inline src/components/Header/index.js]: Why are there two event listeners?

File: src/components/Header/index.js [hunk 3]
```diff
@@ -1,15 +1,32 @@
...
+  }, []);
```
Comment (zzandland) [inline src/components/Header/index.js]: Nice approach

**Comment by zzandland** [comment]
Consider using react context to share the state for mobile view instead of hooks

File: src/components/Header/index.js (hunk 1 above)
Comment (sarahk1280) [reply src/components/Header/index.js]: I will just use react-responsive library that uses media queries instead of using resize listener

**Pull request merged**